src/components/FlyTooltip/FlyTooltip.jsx

```diff
@@ -12,12 +12,14 @@ export default class FlyTooltip extends Component {
 		forceHoverState: PropTypes.bool,
 		hoverIntent: PropTypes.bool,
 		position: PropTypes.oneOf(['top', 'bottom', 'right']),
+		widthIsAuto: PropTypes.bool,
```

Comment:
This naming feels a little awkward at first glance. What about something like `fluid`?

Reply:
I can change it to fluid. The only negative is that `auto` is a concrete concept in CSS and `fluid` is not. But honestly, I consider this to be pretty small and we can document it so it's clear what it does. Fluid it is!